sstables/super_sstable_reader.go

```diff
@@ -23,13 +23,9 @@ func (s SuperSSTableReader) Contains(key []byte) (bool, error) {
 		if err != nil {
 			return false, err
 		}
-		if !keyExist {
-			continue
+		if keyExist {
+			return true, nil
```

Comment:
do you mind to add a unit test for this too? I wonder why this just passes :D

Reply:
I will check if the test cover this

Reply:
added this as part of https://github.com/thomasjungblut/go-sstables/commit/44b5b677dbbeb105f0c0dcbadf86fb82c8dbf410

thanks for fixing this as well, I'll see whether we need some more test coverage around this right now